frontend/src/views/web/challenge/submission.html

```diff
@@ -61,6 +61,14 @@ <h5 class="w-300">
                             <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
                         </button>
                     </li>
+                    <li>Make private submission<br>
```

Comment:
Can we please change it to -- Use `--private` or `--public` flag in the submission command to make the submission private or public respectively.